lib/danger/dangerfile_dsl.rb

```diff
@@ -42,7 +44,9 @@ def fail(message, sticky = true)
       #
       # @param    [String] message
       #           The message to present to the user
-      def warn(message, sticky = true)
+      # @param    [Boolean] sticky
+      #           Whether the message should be kept after it was fixed
+      def warn(message, sticky: true)
```

Comment:
I don't think so, because there're already some named parameters on the codebase